recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
```

Comment:
Well, this is a subtle problem. Basically it appears we cannot build this with the CIs as `git log` cannot run without complaining about the terminal not being fully operational. Not really sure what we want to do about this.